src/lib/components/setlist/EditLineupModal.svelte

```diff
@@ -114,16 +122,36 @@
 	on:click:button--secondary={() => {
 		open = false
 	}}
-	primaryButtonDisabled={modalParticipants.some(
+	on:close={() => {
+		open = false
+	}}
+	primaryButtonDisabled={modalLineup.some(
 		({ participantId, instrument }) =>
 			(participantId && !instrument) || (instrument && !participantId)
 	)}
 	hasScrollingContent
+	hasForm
 	class="editMusiciansOnSongModal"
 >
 	<Form>
 		<Grid>
-			{#each modalParticipants as participant, i}
+			<Row class="row-with-bottom-padding">
+				<Column>
+					<ComboBox
+						titleText="Bandcoach"
+						on:select={(e) => (bandcoachId = e.detail.selectedId)}
+						on:clear={() => (bandcoachId = undefined)}
+						selectedId={bandcoachId}
+						items={participantListItems}
+						shouldFilterItem={(item, value) => {
+							if (!value) return true
+							return item.text.toLowerCase().includes(value.toLowerCase())
+						}}
+					/>
+				</Column>
+			</Row>
+
+			{#each modalLineup as participant, i}
 				<Row>
 					<Column sm={2} md={4} lg={7}>
 						<ComboBox
```

Comment:
you might want to prevent people from entering "Bandcoach" as instrument here as well

Reply:
Good point

Reply:
There is no feedback now though. Would be nice